absl/debugging/internal/stacktrace_riscv-inl.inc

```diff
@@ -44,7 +44,8 @@ template <typename T>
 static inline ptrdiff_t ComputeStackFrameSize(const T *low, const T *high) {
   const char *low_char_ptr = reinterpret_cast<const char *>(low);
   const char *high_char_ptr = reinterpret_cast<const char *>(high);
-  return low < high ? high_char_ptr - low_char_ptr : kUnknownFrameSize;
+  return low < high ? static_cast<size_t>(high_char_ptr - low_char_ptr)
```

Comment:
This doesn't look correct to me. This function returns `ptrdiff_t`. Why is this being cast to `size_t`?

Reply:
I just copu aarch64
https://github.com/abseil/abseil-cpp/blob/917bfee46514a1da47df74265571f47700922c0a/absl/debugging/internal/stacktrace_aarch64-inl.inc#L78

Reply:
done